macros/schema_tests/column_values_basic/expect_column_values_to_have_consistent_casing.sql

```diff
@@ -0,0 +1,36 @@
+{% test expect_column_values_to_have_consistent_casing(model, column_name, display_inconsistent_columns=False) %}
+
+with test_data as (
+
+     select
+     distinct {{ column_name }} as distinct_values
+     from
+     {{ model }}
+     where 1=1
+
+ ),
+ {% if display_inconsistent_columns %}
+ validation_errors as (
+
+    select
+    lower(distinct_values) as inconsistent_columns,
+    count(distinct_values) as set_count_case_insensitive
+    from test_data
+    group by 1
+    having count(distinct_values)>1
```

Comment:
Formatting: we typically do spaces around operator, e.g. `1 = 1` not `1=1`. So, in this case `count(distinct_values) > 1` for better readability.